scripts/helmlint.sh

```diff
@@ -17,9 +17,12 @@ helm lint "${TARGET_DIR}" --set system_domain=example.com
 # Running json schema validator a second time to also check the embedded config files.
 # Done in a separate run because the config files are merged here on top of values.yaml,
 # which will cause replacement of any arrays.
+#
+# NOTE: Excluded api.yaml, as a config file with templating.
```

Comment:
Should this comment be removed now that config files with templating are excluded generically

Reply:
Yes.

Reply:
Not removed, updated, commit 668858cb443f059c793cf7a319961f9397d30c2f